requirements.txt

```diff
@@ -30,20 +30,23 @@ lazy-object-proxy==1.3.1
 MarkupSafe==1.1.1
 mccabe==0.6.1
 mock==2.0.0
-pbr==5.1.3
 oauthlib==3.0.1
+pbr==5.1.3
+psycopg2==2.7.7
```

Comment:
@BrianSerem, does this dependency contribute to this PR?

Reply:
Yes, it was inside one of the packages as a dependency, it came up after running `pip freeze > requirements.txt`

Reply:
Do we need it too?